lms/views/admin.py

```diff
@@ -0,0 +1,91 @@
+from pyramid.httpexceptions import HTTPFound
+from pyramid.view import forbidden_view_config, view_config, view_defaults
+
+from lms.security import Permissions
+
+
+@view_defaults(request_method="GET")
+class AdminViews:
+    def __init__(self, request):
+        self.request = request
+        self.application_instance = request.find_service(name="application_instance")
+
+    @view_config(
+        route_name="admin.index",
+        renderer="lms:templates/admin/index.html.jinja2",
+        permission=Permissions.ADMIN,
+    )  # pylint: disable=no-self-use
+    def index(self):
+        return {}
+
+    @view_config(
+        route_name="admin.installations",
+        renderer="lms:templates/admin/installations.html.jinja2",
+        permission=Permissions.ADMIN,
+    )  # pylint: disable=no-self-use
+    def installations(self):
+        return {}
+
+    @view_config(
+        route_name="admin.installations",
+        permission=Permissions.ADMIN,
+        request_method="POST",
+    )
+    def find_installation(self):
+        if "query" not in self.request.params:
+            self.request.session.flash("Missing mandatory 'query'", "errors")
+            return HTTPFound(location=self.request.route_url("admin.installations"))
```

Comment:
We're probably okay to just `raise httpexceptions.HTTPBadRequest()` here (without the flash message), in practice only our own form ever `POST`'s to this view and it does always include `query`